packages/react-components/src/PasswordStrength.tsx

```diff
@@ -1,52 +1,78 @@
 // Copyright 2017-2020 @polkadot/react-components authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+import { useTranslation } from '@polkadot/react-components/translate';
 import React from 'react';
 import styled from 'styled-components';
+import strengthTester from 'owasp-password-strength-test';
+
+const MAX_STRENGTH = 7; // equal to number of password tests in owasp strength tester
 
 interface Props {
   className?: string;
   value: string;
 }
 
 function calcStrength (password: string): number {
-  // at least 8 characters
-  return (/.{8,}/.test(password) ? 1 : 0) * (
-    // bonus if longer
-    (/.{12,}/.test(password) ? 1 : 0) +
-    // a lower letter
-    (/[a-z]/.test(password) ? 1 : 0) +
-    // a upper letter
-    (/[A-Z]/.test(password) ? 1 : 0) +
-    // a digit
-    (/\d/.test(password) ? 1 : 0) +
-    // a special character
-    (/[^A-Za-z0-9]/.test(password) ? 1 : 0)
-  );
+  strengthTester.config({
+    allowPassphrases: true,
+    maxLength: 128,
+    minLength: 8,
+    minPhraseLength: 20
+  });
```

Comment:
This should be done in the file as a global, no need to set it each time - once set, it is used.